reproducer.yml

```
@@ -4,6 +4,35 @@
   hosts: "{{ cifmw_target_host | default('localhost') }}"
   gather_facts: true
   pre_tasks:
+    # This allows to create a custom scenario file that would
+    # just override some parts of an existing one.
+    # For example, you create a DT based on the va-hci.yml.
+    # You will then just add
+    # `cifmw_parent_scenario: scenarios/reproducers/va-hci.yml`
+    # in the file, and override/extend the scenario to match your
+    # needs.
+    # The parameter also supports a list of files if needed.
+    #
+    # File path are relative to the root of the ci-framework repository.
+    - name: Inherit from parent parameter file if instructed
+      vars:
+        _file_list: >-
+          {% if cifmw_parent_scenario is defined and
+             cifmw_parent_scenario is string -%}
+             {{ [cifmw_parent_scenario] }}
+          {% elif cifmw_parent_scenario is defined and
+             cifmw_parent_scenario is not string and
+             cifmw_parent_scenario is iterable -%}
+             {{ cifmw_parent_scenario }}
+          {% else %}
+          {{ [] }}
+          {% endif %}
+      when:
+        - item is exists
```

May be better we should fail here instead of silently skipping the include?